js-exercises/drop-elements/dropElements.js

```diff
@@ -0,0 +1,12 @@
+function dropElements(elements, predicate) {
+  let validElements = [];
+  
+  for(let i = 0; i < elements.length; i++) {
```

Comment:
best practice is to calculate length once for array , something like below sample
` const elementLen =  elements.length;
  for(let i = 0; i <elementLen; i++) {`